lib/process-request.js

```diff
@@ -16,11 +16,17 @@ module.exports = (req, options = {}) => {
     req.headers = removeOtherProperties(req.headers, options.whitelist);
   }
 
-  let mimeType = 'application/json';
+  // parse mimetype from content-type header, default to JSON
+  const postData = { mimeType: 'application/json' };
   try {
-    mimeType = contentType.parse(req).type;
+    postData.mimeType = contentType.parse(req).type;
   } catch (e) {} // eslint-disable-line no-empty
 
+  // Per HAR, we send JSON as postData.text, not params.
+  if (postData.mimeType === 'application/json') {
+    postData.text = JSON.stringify(req.body);
+  } else postData.params = objectToArray(req.body || {});
```

Comment:
```suggestion
  } else {
    postData.params = objectToArray(req.body || {});
  }
```

Can you also add some tests for when `params` might be present? Looks like only tests that exist are for `text`.

Reply:
There’s one right here! https://github.com/readmeio/readme-node/blob/989a22d0d19d6ef6cd218b0da7911cdb873f747a/test/process-request.test.js#L235

Reply:
Ah shit sorry, I was just looking at the PR diff not the file itself.